scripts/eurostat/health_determinants/Alcohol_Consumption/process.py

```diff
@@ -0,0 +1,749 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script Load the datasets, cleans it
+and generates cleaned CSV, MCF, TMCF file
+"""
+from sys import path
+# For import util.alpha2_to_dcid
+path.insert(1, '../../../../')
+
+import os
+import pandas as pd
+import numpy as np
+from util.alpha2_to_dcid import COUNTRY_MAP
+from absl import app
+from absl import flags
+
+FLAGS = flags.FLAGS
+default_input_path = os.path.dirname(
+    os.path.abspath(__file__)) + os.sep + "input_files"
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+#health_determinant_eurostat_alcoholconsumption_education_attainment
+def hlth_ehis_al1e(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Please have more descriptive function name and avoid function name same as file name.

Reply:
Done the changes